src/components/ha-humidifier-control.js

```diff
@@ -0,0 +1,142 @@
+import "@polymer/iron-flex-layout/iron-flex-layout-classes";
+import "./ha-icon-button";
+import { html } from "@polymer/polymer/lib/utils/html-tag";
+/* eslint-plugin-disable lit */
+import { PolymerElement } from "@polymer/polymer/polymer-element";
+import { EventsMixin } from "../mixins/events-mixin";
+
+/*
+ * @appliesMixin EventsMixin
+ */
+class HaHumidifierControl extends EventsMixin(PolymerElement) {
```

Comment:
We don't accept new `PolymerElement`s please convert this to `LitElement`.

Reply:
Done